ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Hash.java

```diff
@@ -69,16 +69,23 @@ public static Hash fromHexStringLenient(final String str) {
   }
 
   public static Hash fromPlugin(final org.hyperledger.besu.plugin.data.Hash blockHash) {
-    return blockHash instanceof Hash ? (Hash) blockHash : wrap(Bytes32.fromPlugin(blockHash));
+    return blockHash instanceof Hash
+        ? (Hash) blockHash
+        : wrap(Bytes32.wrap(blockHash.getByteArray()));
   }
 
   @Override
   public byte[] getByteArray() {
-    return super.getByteArray();
+    return toArrayUnsafe();
   }
 
   @Override
   public String getHexString() {
-    return super.getHexString();
+    return toHexString();
+  }
+
+  @Override
```

Comment:
As the only thing that `size()` is doing is delegating to it's parent, isn't this method redundant (i.e. could be deleted?)

Reply:
fixed, thanks.